### DISCUSSION TITLE
Support for noweb syntax

### DISCUSSION THREAD

**Discussion by refaelsh** [nvim-orgmode/orgmode#376] [Q&A] [answered]
I was wandering, does this plugin support the [noweb](https://orgmode.org/manual/Noweb-Reference-Syntax.html) syntax?

Thanks.

**Comment by jgollenz** [comment, marked as answer]
Not that I'm aware of. Feel free to open an issue :+1:

**Comment by refaelsh** [comment]
Done. Here it is: https://github.com/nvim-orgmode/orgmode/issues/377.
